packages/menus/src/containers/MenuContainer.js

```diff
@@ -195,18 +195,38 @@ class MenuContainer extends ControlledComponent {
     }
   }
 
+  getDocuments = () => {
+    const doc = getDocument ? getDocument(this.props) : document;
+    const iframes = doc.querySelectorAll('iframe');
+
+    return [...iframes].reduce(
```

Comment:
This _should_ be good, but lets check that this is transpiled into IE11 compatible syntax.

Reply:
Yeah babel handles this fine transforms it to:

```js
return [].concat(_toConsumableArray(iframes)
```

Tested this already in all browsers in the checkbox marked in the description. So IE11 and edge are all good.